lib/collector.rb

```diff
@@ -290,6 +290,7 @@ def process_web(metric)
       hash = { db: db, api: api_type, verb: metric.verb }
       if metric.background
         hash[:type] = "background"
+        hash[:background_type] = metric.background_type if metric.background_type
```

Comment:
hmmmm.. this `if` feels unnecessary since `background_type` will always be there if `background` is there, right? (I'm assuming it's based off https://github.com/discourse/discourse/pull/16037/files)

Also, should it be `metric.is_background`?

Reply:
> this if feels unnecessary since background_type will always be there if background is there, right?

That will be true once I merge https://github.com/discourse/discourse/pull/16037, and the site is running on tests-passed. But we need this plugin to keep working on older versions of Discourse, including `stable`, so the `if metric.background_type` is essentially checking whether we're running a recent version of core